kuma/static/js/analytics.js

```diff
@@ -107,16 +118,16 @@
 
         trackLink: function(event, url, data) {
             // ctrl or cmd click or context menu
-            var newTab = (event.metaKey || event.ctrlKey || event.type == 'contextmenu');
+            var newTab =
```

Comment:
Yeah, it is because of the line length that prettier is doing that formatting. I will have prettier ignore this line.